tests/desktop/developer_hub/test_developer_hub_submit_apps.py

```diff
@@ -1,5 +1,5 @@
 # coding: utf-8
-#!/usr/bin/env python
+# !/usr/bin/env python
```

Comment:
This shouldn't have a space in it. I think it should be the first line in the file also.

```py
#!/usr/bin/env python
# coding: utf-8